core/endpoint.go

```diff
@@ -104,6 +104,12 @@ type Endpoint struct {
 	// Interval is the duration to wait between every status check
 	Interval time.Duration `yaml:"interval,omitempty"`
 
+	// FailedInterval is the duration we'll use to replace Interval when an endpoint stauts check is failed
+	FailedInterval time.Duration `yaml:"failed-interval,omitempty"`
+
+	// IntervalChanged is whether the endpoint's Interval has changed before
+	IntervalChanged bool `"omitempty"`
```

Comment:
This bool doesn't seem to be useful? Is starts as `false` and is set to `true` if there's an error. Maybe i'm missing something?

Reply:
@heitorPB I'll make sure what is it for and fix it later thank you